app/javascript/widget/components/UserMessageBubble.vue

```diff
@@ -2,17 +2,19 @@
   <div
     v-dompurify-html="formatMessage(message, false)"
     class="chat-bubble user"
+    :class="$dm('', 'has-dark-mode')"
```

Comment:
@nithindavid Do we need the dark mode mixin here since we are not using `has-dark-mode` for the user bubble?

Reply:
Makes sense, we can remove it